uniflow/op/extract/load/html_op.py

```diff
@@ -23,9 +36,7 @@ def __call__(self, nodes: Sequence[Node]) -> Sequence[Node]:
         for node in nodes:
             value_dict = copy.deepcopy(node.value_dict)
             if "url" in value_dict:
-                import requests  # pylint: disable=import-outside-toplevel
-
-                resp = requests.get(url=value_dict["url"], timeout=300)
+                resp = self._requests_client.get(url=value_dict["url"], timeout=300)
                 text = resp.text
             else:
```

Comment:
Can we update this `else` to check `filename` and then add a else case to log regarding a specific extract format is not supported. This should help improved the readability.